src/Mod/Sketcher/Gui/DrawSketchControllableHandler.h

```diff
@@ -102,6 +102,17 @@ class DrawSketchControllableHandler
         DSDefaultHandler::finish();
         return true;
     }
+
+    bool shouldDrawPositionAtCursor() const override

```

Comment:
This override is the result of a different decision: extending the interface so that every single handler can have a saying on drawing or not drawing. I will comment on this one on the source.